scrabble.js

```diff
@@ -1,8 +1,129 @@
-var Scrabble = function() {};
+var Scrabble = function() {
+};
+
+Scrabble.prototype.score = function (word) {
+  this.word = word.toLowerCase().split("");
+  var score = 0;
+  for (var i = 0; i < this.word.length; i++) {
+    switch (this.word[i]) {
+      case "a": case "e": case "i": case "o": case "u": case "l": case "n": case "r": case "s": case "t":
```

Comment:
This is definitely not the most efficient way to score this - can you use some sort of object to simplify and DRY this up?